### PR TITLE
[ACL] Add support to match on Tunnel Termination

### PR DESCRIPTION
**Pull request by vivekrnv**
**What I did**

Add support to match an ACL rule on Tunnel Termination Flag. 

**Why I did it**

**How I verified it**

Added UT and verified create_acl_entry with this attribute is working properly

```
vkarri@0f4071a10800:/sonic/src/sonic-swss/tests/mock_tests$ ./tests --gtest_filter="*AclTableType_Configuration*"
Running main() from /build/googletest-YnT0O3/googletest-1.10.0.20201025/googletest/src/gtest_main.cc
Note: Google Test filter = *AclTableType_Configuration*
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from AclOrchTest
[ RUN      ] AclOrchTest.AclTableType_Configuration
[       OK ] AclOrchTest.AclTableType_Configuration (12 ms)
[----------] 1 test from AclOrchTest (12 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (12 ms total)
[  PASSED ] 1 test.

```

**Details if related**

### CODE REVIEW

File: orchagent/aclorch.cpp [hunk 1]
```diff
@@ -807,6 +808,13 @@ bool AclRule::validateAddMatch(string attr_name, string attr_value)
         {
             return false;
         }
+        else if (attr_name == MATCH_TUNNEL_TERM)
+        {
+            if (attr_value != "true")
+            {
+                matchData.enable = false;
```
Comment (oleksandrivantsiv) [inline orchagent/aclorch.cpp:815]: matchData.enable = (attr_value == "true")?

Comment (vivekrnv) [reply orchagent/aclorch.cpp:815]: This is already set here. https://github.com/vivekrnv/sonic-swss/pull/32/files#diff-d97175d30783db8709cc74eb41e215bf958bcf304be11d836830b03cd29e917cR803

Comment (oleksandrivantsiv) [reply orchagent/aclorch.cpp:815]: I see that. But still, I suggest making the code compact and removing unneeded lines.

Comment (vivekrnv) [reply orchagent/aclorch.cpp:815]: Hmm, will update

Comment (vivekrnv) [reply orchagent/aclorch.cpp]: updated

**Review by dgsudharsan — APPROVED** [review]

**Pull request closed without merging**